sympy/geometry/polygon.py

```diff
@@ -403,7 +403,7 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
-    def second_moment_of_area(self,p=None):
+    def second_moment_of_area(self, p = None):
```

Comment:
There should be a space after comma but no spaces around the equality sign as a default parameter value is given. (PEP 8: Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.)

Reply:
@rushyam I recommend setting up a linter in your text editor. Look up pyflakes, pep8, flake8, pylint, etc. Then we want have to go back an forth on all these. Also read this: https://www.python.org/dev/peps/pep-0008/

Reply:
thankyou @jksuom and @moorepants. I installed pylint now it just giving message related to inappropriate name in Polygon class.I think it is not a thing to worry.